django_socio_grpc/protobuf/proto_classes.py

```diff
@@ -405,6 +508,31 @@ def _from_serializer_method_field(
         )
 
 
+@dataclass
+class ProtoEnum:
+    enum: Enum = None
+    wrap_in_message: bool = False
+
+    def __eq__(self, other):
+        if not isinstance(other, ProtoEnum):
+            return NotImplemented
+
+        # Enums may be created on the fly and so we need to compare the enum members
+        self_enum = (self.enum.__name__, tuple(self.enum.__members__.items()))
+        other_enum = (other.enum.__name__, tuple(other.enum.__members__.items()))
+
+        return self_enum == other_enum and self.wrap_in_message == other.wrap_in_message
+
+    def __hash__(self):
+        enum_hash = hash((self.enum.__name__, tuple(self.enum.__members__.items())))
+        return hash((enum_hash, self.wrap_in_message))
+
+
+@dataclass
+class ProtoExtraTools:
```

Comment:
It's a way to group extra data that can be defined in the proto spec.
Other choice can be imoort at top of file, extensions, oneof, nested message. 
Of course each could be it own property. I thought this will be cleaner this way

Reply:
Will ProtoExtraTools be usefull in the near future ? 
It's easier to print all enums using ProtoExtraTools as it require no pre-processing in generators.py but it's maybe adding useless complexity in the structure for what it is

Reply:
It depend of @legau on import at top of file, extensions, oneof, nested message.
If he see an other way to do them we can remove it.

Reply:
This may all be made into class attributes when we support them

Reply:
There is no reason why `fields` should be an attribute and not `enums`, they're both properties of the message

Reply:
import are not. But agree with the logic it's not such a big deal